runorders.cpp

```diff
@@ -3027,14 +3037,14 @@ int Game::DoGiveOrder(ARegion *r, Unit *u, GiveOrder *o)
 	}
 
 	if (o->type == O_TAKE) {
+		// TODO JSON
```

Comment:
TODO?

Reply:
Only if we want a specialized TAKE Json report. It will show up now as plain text.